pages/design/ui-patterns/button-usage.mdx

```diff
@@ -0,0 +1,117 @@
+
+#### Buttons
+
+At GitHub, buttons are a fundamental building block of our products. The following table catalogues a quick glance at how we use buttons at GitHub: 
+
+| Button | Visual | Usage | Notes | Modifiers |
+| --- | ------------- | --- | --- | --- |
+| Secondary | ![btn](https://user-images.githubusercontent.com/24916540/52498774-86e4c200-2b8e-11e9-953b-e0e7a70493ea.png) | For secondary or less important actions | Should be considered as the go-to style when designing new buttons | Octicons, Size, Full-width, Button-Groups, Counter |
+| Outline | ![btn-outline](https://user-images.githubusercontent.com/24916540/52498769-864c2b80-2b8e-11e9-984a-d5b88c846b74.png) | For general actions, navigation, and filtering | Should be used independently and not in conjunction with other button styles | Octicons, Size, Full-width, Button-Groups, Counter |
+| Primary | ![btn-primary](https://user-images.githubusercontent.com/24916540/52498773-86e4c200-2b8e-11e9-804f-146d721d5191.png) | To indicate the primary action on a page or view | Avoid using multiple primary buttons in a single view | Octicons(limited), Size, Full-width |
+| Danger | ![btn-danger](https://user-images.githubusercontent.com/24916540/52498772-864c2b80-2b8e-11e9-8eeb-bef8729c1fd1.png) | To indicate potentially dangerous, or destructive actions | Only use this button when necessary | Octicons(limited), Size, Full-width|
+
+## Usage
+
+### Use Sentence Case
+
+At GitHub, our buttons are all sentence-cased. Please refrain from deviating from this standard.
+
+<Flex>
+  <Box width='100%' mr='2'>
+    <BorderBox border='1px solid' borderColor='green.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/52501276-d3330080-2b94-11e9-954e-38c0f09f8ca1.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='green.5'>Do:</Text> Use sentence case 
+  </Box>
+  <Box width='100%' mr='2'>
+    <BorderBox border='1px solid' borderColor='red.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/52501272-d3330080-2b94-11e9-8829-7d12aae36ba1.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='red.5'>Don't:</Text> Use all-caps or other text formats
+  </Box> 
+</Flex>
+
+### Keep Button Text Concise
+When designing buttons, keep the text as short but descriptive as possible. Buttons are to be understood by their type and text; if further clarity is required, consider adding an octicon or adding a text description. Never wrap text within buttons. 
+
+### Adaptable Full-Width 
+In certain scenarios, it may be appropriate for a button’s width to be extended to fill their parent container. Use this style sparingly, but some general, appropriate use-cases include:
```

Comment:
```suggestion
In certain scenarios, it may make sense to use full-width buttons that extend to the width of their container. Use this style sparingly, but some appropriate use-cases include:
```